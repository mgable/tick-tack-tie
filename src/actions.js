/*
 * action types
 */

export const SELECT_SQUARE = 'SELECT_SQUARE'

/*
 * action creators
 */

export function selectSquare(idx) {
  return { type: SELECT_SQUARE, idx }
}
