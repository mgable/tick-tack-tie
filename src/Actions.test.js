
import toJson from 'enzyme-to-json';

import { rewindHistory, selectSquare } from './actions';



describe('Actions', () => {
	test('correctly makes rewind history', () => {
		expect(rewindHistory()).toMatchSnapshot();
	});

	test('correctly makes select Square', () => {
		expect(selectSquare(1)).toMatchSnapshot();
	})
	
});
