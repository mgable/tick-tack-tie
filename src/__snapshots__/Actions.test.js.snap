// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Actions correctly makes rewind history 1`] = `
Object {
  "type": "REWIND_HISTORY",
}
`;

exports[`Actions correctly makes select Square 1`] = `
Object {
  "idx": 1,
  "type": "SELECT_SQUARE",
}
`;
